tensilelite/Tensile/KernelWriter.py

```diff
@@ -2593,6 +2591,11 @@ def _loopBody( self, kernel, tensorParametersA, tensorParametersB, pack, lc, loo
           pointerLWCode.addComment1("local write swap offsets b")
           pointerLWCode.add(self.localWriteSwapOffsets(kernel, expand, tensorParametersB))
 
+        # DTL: Wait for previous set of global reads to complete
+        if kernel["DirectToLdsA"] or kernel["DirectToLdsB"]:
```

Comment:
I think this should be the same location as before your change.
I think we need this condition.
  if u == localWriteEndIter+1

Is there any reason you moved the code here?

Reply:
Ooops, you're right, moved to the waitcnt to the previous location. I just realized the previous waitcnt for DTL was adding to `module`.. changed to `waitLWCode` instead.

Reply:
Thanks for your confirmation.
That is what I said in my previous comment.
The current change looks good.
I do not have any further comment for now.